_posts/2020-18-06-Triplestores-Onotologies-Reasoning.md

```diff
@@ -54,12 +61,14 @@ describes various means and methods of representing knowledge in a
 computer.  These terms are often used interchangeably and can cause
 confusion if a reader is unfamiliar with their use.
 
+[post]: {% post_url 2020-06-20-Basic-Querying.md%}
```

Comment:
It's not clear to me why this is here but the url is now 2020-06-20-Some-Examples-of-Querying.md